src/index.html

```diff
@@ -10,7 +11,53 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      name="bigForm"
+    >
+
+      <label class="box">
```

Comment:
```suggestion
    <form
      action="#"
      method="post"
      data-qa="big"
      name="bigForm"
    >
      <label class="box">
```

No need to add blank line between parent and child elements

Reply:
ок